elixir-cypress-ci/Dockerfile

```diff
@@ -17,8 +17,10 @@ RUN mkdir -p /etc/apt/keyrings && \
 
 RUN apt update -y
 
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - &&\
```

Comment:
I went for node 18 but I'm not sure if that was right? It's what the platform has in it's tool-versions file